package.json

```diff
@@ -16,14 +16,14 @@
     "build:tokens": "yarn --cwd ./packages/design-system-tokens build themes scss && yarn --cwd ./packages/design-system-tokens build themes css-vars && yarn --cwd ./packages/design-system-tokens dist",
     "deploy-demo": "ts-node ./scripts/deployDemo.ts",
     "lint": "yarn lint:stylelint && yarn lint:eslint",
-    "lint:eslint": "npx eslint ./packages/",
-    "lint:stylelint": "npx stylelint ./packages/**/*.scss --custom-formatter=node_modules/stylelint-formatter-pretty",
+    "lint:eslint": "yarn eslint ./packages/",
+    "lint:stylelint": "yarn stylelint ./packages/**/*.scss --custom-formatter=node_modules/stylelint-formatter-pretty",
```

Comment:
For some reason the `npx` command was failing in Jenkins in the new docker container, so I just changed it here.